### PR TITLE
minor corrections and improvements

### PR DESCRIPTION
**Pull request by lyakh**
Mostly cosmetic improvements

### CODE REVIEW

**Review by dbaluta — APPROVED** [review]

File: src/platform/library/platform.c [hunk 1]
```diff
@@ -61,3 +61,17 @@ int platform_init(struct sof *sof)
 
 	return 0;
 }
+
+#ifdef __ZEPHYR__
+/* Dummies for unsupported architectures */
```
Comment (lgirdwood) [inline src/platform/library/platform.c:66]: Should call this something else than dummy

**Review by lgirdwood — COMMENTED** [review]
LGTM, but CI build failure in testbench.

**Comment by lyakh** [comment]
AFAICS CI errors are unrelated or can / should be ignored, e.g. I hope we can accept a commit with subject "remove superfluous parentheses" with no commit message. I really would not know how else to describe what the patch does :-)

**Comment by lgirdwood** [comment]
Internal CI has a 

> AFAICS CI errors are unrelated or can / should be ignored, e.g. I hope we can accept a commit with subject "remove superfluous parentheses" with no commit message. I really would not know how else to describe what the patch does :-)

Ack, looks like WHL DUT HW issue and unrelated as other WHL DUT is pasing.

File: src/platform/library/platform.c (hunk 1 above)
Comment (lgirdwood) [reply src/platform/library/platform.c:66]: Can you use the word "stub" here instead of dummies.

Comment (lyakh) [reply src/platform/library/platform.c]: @lgirdwood done

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**